pyproject.toml

```
@@ -25,6 +25,11 @@ dependencies = [
     "wcwidth",
 ]
 
+[project.urls]
+homepage = "https://github.com/prompt-toolkit/python-prompt-toolkit"
+source = "https://github.com/prompt-toolkit/python-prompt-toolkit"
+documentation = "https://python-prompt-toolkit.readthedocs.io/en/stable/"
```

```suggestion
Homepage = "https://github.com/prompt-toolkit/python-prompt-toolkit"
Documentation = "https://python-prompt-toolkit.readthedocs.io/en/stable/"
```
A minor suggestion. Not sure there is a need to duplicate urls.

---

They show under different headings in PyPI hence I included both so you get both links. Happy to adjust though if you would rather simplify?

---

True. Though I wonder how useful it would be to have to different headings link to the same page.

I'd also suggest to capitalize the keys.

---

Have updated accordingly